torch/nn/modules/normalization.py

```diff
@@ -90,9 +90,10 @@ class LayerNorm(Module):
     .. math::
         y = \frac{x - \mathrm{E}[x]}{ \sqrt{\mathrm{Var}[x] + \epsilon}} * \gamma + \beta
 
-    The mean and standard-deviation are calculated separately over the last
-    certain number dimensions which have to be of the shape specified by
-    :attr:`normalized_shape`.
+    The mean and standard-deviation are calculated over a number of last dimensions
+    specifies by the shape of :attr:`normalized_shape`. For example, if the shape of
+    :attr:`normalized_shape` is 2, then the last two dimensions are used to compute
```

Comment:
nit: I think it's more correct to refer to the "dimension" of `normalized_shape` rather than its shape, as a shape being 2 isn't quite right. Maybe something like:

```
The mean and standard-deviation are calculated over the last `D` dimensions, where `D` is the dimension of :attr:`normalized_shape`.
For example, if :attr:`normalized_shape` is ``(3, 5)`` (a 2-dimensional shape), the mean and standard-deviation are computed over the last 2 dimensions of the input (i.e. ``input.mean((-2, -1))``).
```